trace/HTTPAttributes.md

```diff
@@ -0,0 +1,33 @@
+# HTTP Attributes
+
+This document summarizes HTTP attributes recorded from HTTP
+requests and responses.
+
+HTTP integrations should set the following attributes on the client
+and server spans. For a server, request represents the incoming request.
+For a client, request represents the outgoing request.
+
+All attributes are optional.
+
+| Attribute name       | Description            | Example value                   |
+|----------------------|------------------------|---------------------------------|
+| "http.host"          | Request URL host       | "example.com"                   |
+| "http.method"        | Request URL method     | "GET"                           |
+| "http.path"          | Request URL path       | "/users"                        |
+| "http.user_agent"    | Request user-agent     | "HTTPClient/1.2"                |
+| "http.status"        | Response status code   | 200                             |
```

Comment:
I would name this status_code

Reply:
You were reviewing an old version of the spec, this is renamed.